global_frame: 'map'
use_global_planner: True
num_of_retries: 5
allow_backward_motion: True

# recovery
recovery_enabled: True
recovery_wait_duration: 3.0 # seconds
recovery_motion_duration: 3.0 # seconds
recovery_vel: -0.15

# tolerances
heading_tolerance: 0.5
goal_dist_tolerance: 0.1
goal_theta_tolerance: 0.1
latch_xy_goal: True
dist_between_wp: 1.0
waypoint_dist_tolerance: 0.3
max_safe_costmap_val: 80

# controller params
control_rate: 5.0
p_theta_in_place: 5.0
p_theta: 1.0
p_linear: 1.0
max_theta_vel: 0.5
min_theta_vel: 0.005
max_linear_vel: 0.3
min_linear_vel: 0.1
future_pos_lookahead_dist: 0.4
